sql/item_mods.sql

```diff
@@ -28459,7 +28459,8 @@ INSERT INTO `item_mods` VALUES (15697,20,6); -- WATER_RES: 6
 INSERT INTO `item_mods` VALUES (15697,26,2); -- RACC: 2
 
 -- Sneaking Boots
-INSERT INTO `item_mods` VALUES (15698,1,9); -- DEF: 9
+INSERT INTO `item_mods` VALUES (15698,1,9);   -- DEF: 9
+INSERT INTO `item_mods` VALUES (15698,358,1); -- STEALTH: 1
```

Comment:
well, I'm glad you went with the minimum value since I can't find any data about how much it enhances it by. Also this mod  works even if you don't have the trait already. I don't know if that's also how retail treats it. There is just no info about it. So you are right about some retail checking being needed.

Reply:
The only spot it seems to be used currently is inside of mob_controller.cpp on line 259 where it's calculating currentDistance and adding the mod value to it.

Reply:
right. fun fact: you can add negative amounts to people to make them aggro from shorter distances 😈